pom.xml

```diff
@@ -1,171 +1,261 @@
-    <pluginManagement>
-      <plugins>
-        <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-compiler-plugin</artifactId>
-          <version>${version.maven.compiler.plugin}</version>
-          <configuration>
-            <source>1.8</source>
-            <target>1.8</target>
-            <testSource>1.8</testSource>
-            <testTarget>1.8</testTarget>
-          </configuration>
-        </plugin>
-      </plugins>
-    </pluginManagement>
-  </build>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.2.4.RELEASE</version>
+		<relativePath/>
+	</parent>
+
+	<groupId>eu.europeana.metis</groupId>
+	<artifactId>metis-sandbox</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>metis-sandbox</name>
+	<description>Metis sandbox pilot</description>
+
+	<!-- Github connection -->
+	<scm>
+		<connection>scm:git:https://github.com/europeana/metis-sandbox</connection>
+		<url>https://github.com/europeana/metis-sandbox</url>
+		<tag>HEAD</tag>
+		<developerConnection>scm:git:https://github.com/europeana/metis-sandbox</developerConnection>
+	</scm>
+
+	<distributionManagement>
+		<repository>
+			<id>artifactory.eanadev.org</id>
+			<name>artifactory.eanadev.org-releases</name>
+			<url>http://artifactory.eanadev.org/artifactory/libs-release-local</url>
+		</repository>
+		<snapshotRepository>
+			<id>artifactory.eanadev.org</id>
+			<name>artifactory.eanadev.org-snapshots</name>
+			<url>http://artifactory.eanadev.org/artifactory/libs-snapshot-local</url>
+		</snapshotRepository>
+	</distributionManagement>
+
+	<properties>
+		<version.java>13</version.java>
+		<version.springfox>2.9.2</version.springfox>
+		<version.metis>1.7.0</version.metis>
+		<version.lombok>1.18.12</version.lombok>
+		<version.commons.io>2.6</version.commons.io>
+		<version.maven.compiler.plugin>3.6.1</version.maven.compiler.plugin>
+		<version.coveralls>4.0.0</version.coveralls>
+		<version.surefire.plugin>2.22.1</version.surefire.plugin>
+
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+	</properties>
+
+	<dependencies>
+		<!-- Spring boot -->
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-amqp</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-webflux</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
+		</dependency>
```

Comment:
Spring Boot is a layer on top of spring that enables to develop applications faster, it comes with default config and reduces boilerplate code. The goal is simplifying the development of a Spring application. 
And it is (heavily used)[https://spring.io/microservices] now days , but mostly you are not losing anything from Spring, you can configure anything you want or rely on the defaults, but makes life simpler. From previous job we started using Spring Boot and then again on another team, and while there was skepticism, it works really nice, and it is where I see spring applications going to. So I am not sure there is a reason not to use it but strongly advocate to give it a try
https://www.baeldung.com/spring-vs-spring-boot
https://spring.io/blog/2013/08/06/spring-boot-simplifying-spring-for-everyone/

Reply:
In general we try to avoid too much hidden code from tools like that..
Since we are not going towards spring boot anytime soon in the other metis applications, I would say we stick with consistency between projects as much as we can.
And the embedded server is a nice thing to have for fast deployment but I'm pretty sure Andrea will not be happy if we provide him with something like that..

Reply:
I will leave it for now, since I have already made many changes to the foundations, and I need to merge those in the other branches I have (which the first one maybe is complex), plus also I have more to say about this topic, for which I will schedule a meeting to talk about